index.js

```diff
@@ -33,13 +31,31 @@ var os = require('os'),
  *
  * @param {string} dir Base directory.
  * @param {string} [appname] Optional name to append.
+ * @param {string} [appauthor] Optional author to append.
  * @param {string} [version] Optional version to append.
  * @returns {string} Resulting path
  * @private
  */
-var appendNameVersion = function (dir, appname, version) {
+var appendNameAuthorVersion = function (dir, appname, appauthor, version, includeAppAuthor) {
```

Comment:
Instead of renaming the function and adding a `includeAppAuthor` param, just add `appendNameAuthorVersion` as a new function and leave the old function (and the macOS implementation) alone.

Since this will change the locations of files, this is going to be a breaking change worthy of a 2.0 release. It'd be nice if there was an option to get back to the old behavior, even if that option was deprecated and removed in a 3.0 release. Any thoughts on that?

Reply:
I have addressed all your queries/comments. I have left the current Windows implementation as the default so that it does not result in a breaking change for the location of files. Included the functionality of including app author in paths for Windows using an opt-in config `includeAppAuthor`.